src/components/OpenFoodFactsAddMenu.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <v-menu scroll-strategy="close">
+    <template v-slot:activator="{ props }">
+      <v-btn v-bind="props" size="small" color="primary" prepend-icon="mdi-plus" append-icon="mdi-menu-down">
+        <i18n-t keypath="Common.AddToOFF" tag="span">
+          <template #name>{{ OFF_NAME }}</template>
+        </i18n-t>
+      </v-btn>
+    </template>
+    <v-list>
+      <template v-for="(source, index) in sourceList" :key="source.key">
+        <v-list-item :slim="true" :href="getSourceAddUrlWithLocale(source)" target="_blank">
+          <template v-slot:prepend>
+            <v-icon :icon="source.icon"></v-icon>
+          </template>
+          <i18n-t keypath="Common.AddToOFF" tag="span">
+            <template #name>{{ source.value }}</template>
+          </i18n-t>
+          <template v-slot:append>
+            <v-icon size="small" icon="mdi-open-in-new"></v-icon>
+          </template>
+        </v-list-item>
+        <v-divider v-if="index === 0" class="mt-2 mb-2"></v-divider>
+      </template>
+    </v-list>
+  </v-menu>
+</template>
+
+<script>
+import { mapStores } from 'pinia'
+import { useAppStore } from '../store'
+import constants from '../constants'
+
+export default {
+  props: {
+    product: {
+      type: Object,
+      required: true
+    }
+  },
+  data() {
+    return {
+      OFF_NAME: constants.OFF_NAME,
+      sourceList: constants.PRODUCT_SOURCE_LIST,
+    }
+  },
+  computed: {
+    ...mapStores(useAppStore),
+  },
+  methods: {
+    getSourceUrl(source) {
+      return constants[`${source.key.toUpperCase()}_URL`]
+    },
+    getSourceAddUrlWithLocale(source) {
+      const SOURCE_ADD_URL = `${this.getSourceUrl(source)}/cgi/product.pl?type=search_or_add&action=process&code=${this.product.code}`
```

Comment:
@raphael0202 @teolemon is this the correct url to add a product ?

example : https://world.openproductsfacts.org/cgi/product.pl?type=search_or_add&action=process&code=9782330181055

Reply:
Does it work?

Reply:
I get an error. But the product is actually created. So sort of ^^

Reply:
j'ai remplacé `action=process` par `action=display`